erda-helm/erda/charts/volume-provisioner/templates/volume-provisioner-damonset.yaml

```
@@ -2,22 +2,22 @@ kind: DaemonSet
 apiVersion: apps/v1
 metadata:
   labels:
-    app: erda-volume-provisioner
+    app: volume-provisioner
   name: erda-volume-provisioner
   namespace: {{ .Values.global.namespace | default "erda-system" }}
 spec:
   selector:
     matchLabels:
-      app: erda-volume-provisioner
+      app: volume-provisioner
   updateStrategy:
-    type: {{ .Values.global.updateStrategy.type | default "RollingUpdate" }}
+    type: RollingUpdate
```

Need to be configurable.
```suggestion
    type: {{ .Values.global.updateStrategy.type | default "RollingUpdate" }}
```